### DISCUSSION TITLE
Map overlay hover style

### DISCUSSION THREAD

**Discussion by FilipLeitner** [hslayers/hslayers-ng#3412] [Polls]
### Define map overlay hover styles.
 Map overlay includes `toolbar, ol-controls, custom ol-controls, basemap gallery` (scale and hs-info also but those don't use hover property )

**Comment by jmacura** [comment]
I think hs-info component is a bit too aggressive now. Can we change its background from --ol-background-color
![image](https://user-images.githubusercontent.com/5598693/194436985-1ebb2363-31f4-4a16-9cbf-4b0a9f2db00c.png)
to --ol-partial-background-color ?
![image](https://user-images.githubusercontent.com/5598693/194437167-7e38012e-9b92-40a6-bc06-a654471bbd2e.png)

Or I would make this message a bit less spacious. Like a one-liner next to the scale. Or omit its heading. Now it takes too much attention to my taste...

**Reply by FilipLeitner** [reply]
True, looks better with --ol-partial-background-color. Regarding title, Id be for omitting it for the 'unsaved map' case as its basically just says the same info twice.
